<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NozzlePreview</class>
 <widget class="QWidget" name="NozzlePreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>175</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>175</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QListWidget" name="tabAlerts">
      <attribute name="title">
       <string>Alerts</string>
      </attribute>
     </widget>
     <widget class="QGraphicsView" name="tabCrossSection">
      <attribute name="title">
       <string>Cross Section</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
